docs/mine/lotus/manage-storage-deals.md

```diff
@@ -1,33 +1,139 @@
+- Using the `lotus-miner storage-deals selection` commands.
+
+For example, to disable storage deals:
+
+```sh
+lotus-miner storage-deals selection reject --online
+lotus-miner storage-deals selection reject --offline
+```
+
+The commands above will automatically update the values in the `config.toml` file.
+
+You can verify the current status with:
+
+```sh
+lotus-miner storage-deals selection list
+```
+
+You can re-enable storage deals with:
+
+```sh
+lotus-miner storage-deals selection reset
+# Verify that they have been enabled
+lotus-miner storage-deals selection list
+```
+
+Note that the values above affect to new deals. Ongoing deals will still have to be honored.
+
+## Setting the ask price
+
+One of the most important aspects of accepting new deals will be the conditions and price that miners request. Incoming deals are evaluated on these conditions and automatically accepted or rejected by Lotus miners.
+
+Storage prices (and other conditions) are set with the `lotus-miner storage-deals set-ask` command. For example:
+
+```sh
+lotus-miner storage-deals set-ask \
+  --price 100000000000 \
+  --verified-price  100000000000 \
+  --min-piece-size 56KiB \
+  --max-piece-size 32GB
+```
+
+The above command sets the price for deals to _100000000000 attoFIL_ (100 nanoFIL) per Gibibyte per Epoch. That means that a client will have to pay 100 nanoFIL every 30 seconds for each Gibibyte stored. If the client wants 5GiB stored over the course of a week, the total price will be: \_5GiB \* 100nanoFIL/GiB_Epoch \* 20160 Epochs = 10080 microFIL`.
+
+The command also serves to set the minimum and maximum deal sizes. Be sure to check `lotus-miner storage-deals set-ask --help` to see all options.
+
+You can display the miner's current ask price with:
+
+```sh
+lotus-miner storage-deals get-ask
+```
+
+Lotus clients can request the miner price as well with:
 
 ```sh
-lotus-miner storage-deals --help
+lotus client query-ask <minerID>
 ```
 
-For example, `lotus-miner storage-deals set-ask` allows to set the price for storage that your miner uses to respond ask requests from clients.
+## Listing current deals
 
-If deals are ongoing, you can check the data transfers with:
+Current deals and their current state can be obtained with:
 
 ```sh
-lotus-miner data-transfers list
+lotus-miner storage-deals list -v
 ```
 
-:::tip
-Explore the `lotus-miner` CLI. Every command is self-documented and takes a `--help` flag that offers specific information about it.
-:::
+The list displays when the deal was created, the DataCID that is being stored, the wallet address of client that submitted it, the size and the duration in epochs (30 seconds per epoch).
+
+## Using filters to limit deals
+
+Lotus Miners may want to customize on what conditions deals are accepted through custom logic. This can be achieved by providing an external program or script in the `Filter` option of the `[Dealmaking]` [section in the configuration](miner-configuration.md).
```

Comment:
```suggestion
Lotus Miners may want to customize the conditions that deals are accepted. This can be achieved by providing an external program or script in the `Filter` option of the `[Dealmaking]` [section in the configuration](miner-configuration.md).
```